plcc/src/proteinstructure/Residue.java

```diff
@@ -299,7 +294,7 @@ public Boolean interchainContactPossibleWithResidue (Residue r) {
             atomRadius = Settings.getInteger("plcc_I_lig_atom_radius");
         }
         else {
-            atomRadius = 40; //Settings.getInteger("plcc_I_atom_radius");
+            atomRadius = 40; //Settings.getInteger("plcc_I_aa_atom_radius");
```

Comment:
I do not know why we use a hard coded value here. Would be great if you could change it to use the setting instead! :)